pkgs/llvm_cheriot.nix

```diff
@@ -0,0 +1,28 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+{stdenv, fetchFromGitHub, lib, cmake, ninja, lld, python3}:
+stdenv.mkDerivation rec {
+  name = "llvm-cheriot";
```

Comment:
Instead of name, this should be using pname and version (name is being phased out)